CODE_OF_CONDUCT.md

```diff
@@ -33,8 +33,8 @@ that we ask all community members to adhere to:
 -   be friendly and patient,
 -   be considerate,
 -   be kind,
--   be careful in the words that we choose,
--   when we disagree, we try to understand why, and
+-   be careful in the words that you choose,
+-   when we disagree, try to understand why, and
```

Comment:
I think I like the "we"s here -- it suggests more of a "collaborating together on our culture" mindset. I wonder if we should change the bullets below to match these ones rather than the other way around.